packages/core/src/request.ts

```diff
@@ -31,7 +31,7 @@ function enhanceEventWithSdkInfo(event: Event, sdkInfo?: SdkInfo): Event {
   return event;
 }
 
-/** Creates a SentryRequest from an event. */
+/** Creates a SentryRequest from a Session. */
```

Comment:
This type of small fix is very easy to ship in a trivial PR, don't need to block waiting for this large one to ship, nor does it need to distract us during review.

Reply:
Sure agreed